.github/workflows/ci.yml

```diff
@@ -128,7 +132,7 @@ jobs:
           go install github.com/pelletier/go-toml/cmd/tomljson@latest
           echo "$HOME/go/bin" >> $GITHUB_PATH
 
-      - run: ci/test_all_features.sh
+      - run: ci/test_all_features.sh ${{ matrix.std }}
```

Comment:
I think it's quite nice to see if both no_std and std are broken with a quick glance at the job output. Also it would double our CI time, since it would be the longest running job by ~2x the previous one. So I changed this back to use a matrix. I did make the matrix a bit more descriptive though, so that the job output is clearer.

Reply:
@JelteF you should edit branch protection rules, because now CI awaits `
test features` job to complete, while we have  `
test features (std)` and  `
test features (no_std)` only now.

Reply:
Done. If you approve the PR it should merge automatically.